libs/utils/src/http/endpoint.rs

```diff
@@ -129,7 +159,9 @@ pub fn make_router() -> RouterBuilder<hyper::Body, ApiError> {
         .middleware(Middleware::post_with_info(
```

Comment:
Just noticed these two, or at least the logger, it seems quite unfortunate but this explains why it is not within the span.

Should we move it within the span on this PR?

Comment is about line L128 R158 `Middleware::post_with_info(logger)` which I assume is the culprit for spanless `GET /v1/foobar 200 OK`.

Reply:
Handled in e5ece3f4a1d4a75e531729fb8122cae9aa12e80f

Reply:
After the above comment about `span!` placement, something clicked and I've realized how it could be done indeed.

I've not changed the original logic, that did `debug` for `GET` requests and did `info` for other requests and non-200 responses, but not sure how good that is.

Now the logs look like

```
2023-03-08T13:09:13.682288Z DEBUG parsed 3 headers
2023-03-08T13:09:13.682302Z DEBUG incoming body is empty
2023-03-08T13:09:13.682443Z DEBUG request{method=GET path=/v1/status request_id=142ecda3-5274-419b-847d-5a794d2527a6}: Handling request
2023-03-08T13:09:13.682469Z  INFO request{method=GET path=/v1/status request_id=142ecda3-5274-419b-847d-5a794d2527a6}: Status check called
2023-03-08T13:09:13.682482Z DEBUG request{method=GET path=/v1/status request_id=142ecda3-5274-419b-847d-5a794d2527a6}: Status check debug message
2023-03-08T13:09:13.682528Z DEBUG request{method=GET path=/v1/status request_id=142ecda3-5274-419b-847d-5a794d2527a6}: Request handled, status: 200 OK
2023-03-08T13:09:13.682593Z DEBUG flushed 228 bytes
2023-03-08T13:09:13.682700Z DEBUG read eof
```